ansible-runner/context/app/project/roles/mirror_appliance/tasks/secret-vars.yaml

```diff
@@ -2,5 +2,5 @@
   set_fact:
     secret_key: "{{ lookup('community.general.random_string', length=48, base64=True) }}"
     database_secret_key: "{{ lookup('community.general.random_string', length=48, base64=True) }}"
-    pgdb_password: "{{ lookup('community.general.random_string', length=24, base64=True) }}"
```

Comment:
ah, I went with base64 thinking [a-z[A-Z][0-9] would be safe enough for this automation and forgot about the couple special characters it allows.